thrimshim/thrimshim/main.py

```diff
@@ -94,6 +125,13 @@ def update_row(ident, new_row):
 	for extra in extras:
 		del new_row[extra]
 
+	#validate title length - YouTube titles are limited to 100 characters, 9 reserved for "DBXXXX - ".
+	if(len(new_row['video_title']) > 91):
```

Comment:
style nitpick - spaces, no outermost parens
```suggestion
	if len(new_row['video_title']) > 91:
```